src/package/abi/Microsoft.Windows.AbiWinRT.targets

```diff
@@ -6,8 +6,493 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </PropertyGroup>
 
-</Project>
+    <PropertyGroup>
+        <AbiWinRTVerbosity Condition="'$(AbiWinRTVerbosity)' == ''">normal</AbiWinRTVerbosity>
+        <AbiWinRTCommandVerbosity Condition="'$(AbiWinRTVerbosity)' == 'high'">-verbose</AbiWinRTCommandVerbosity>
+        <AbiWinRTProjectWinMD>$(OutDir)$(RootNamespace).winmd</AbiWinRTProjectWinMD>
+        <AbiWinRTMergedDir>$(IntDir)Merged\</AbiWinRTMergedDir>
+        <AbiWinRTUnmergedDir>$(IntDir)Unmerged\</AbiWinRTUnmergedDir>
+        <AbiWinRTSkipUnchangedFiles Condition="'$(AbiWinRTSkipUnchangedFiles)' == ''">true</AbiWinRTSkipUnchangedFiles>
+        <AbiWinRTUseHardlinksIfPossible Condition="'$(AbiWinRTUseHardlinksIfPossible)' == ''">false</AbiWinRTUseHardlinksIfPossible>
+        <AbiWinRTPath Condition="'$(AbiWinRTPath)'==''">$(MSBuildThisFileDirectory)..\..\bin\</AbiWinRTPath>
+
+        <!-- Default modern idl semantics to false considering that most users probably have not made the switch to the modern idl syntax -->
+        <AbiWinRTModernIDL Condition="'$(AbiWinRTModernIDL)'==''">false</AbiWinRTModernIDL>
+
+        <!-- Only generate the platform and component projection if using modern idl. Otherwise, allow the legacy
+             toolchain to kick in, as well as use the existing references from the Windows SDK -->
+        <AbiWinRTEnableComponentProjection Condition="'$(AbiWinRTEnableComponentProjection)' == '' and '$(AbiWinRTModernIdl)'!='false'">true</AbiWinRTEnableComponentProjection>
+        <AbiWinRTEnablePlatformProjection Condition="'$(AbiWinRTEnablePlatformProjection)' == '' and '$(AbiWinRTModernIdl)'!='false'">true</AbiWinRTEnablePlatformProjection>
+        <AbiWinRTEnableReferenceProjection Condition="'$(AbiWinRTEnableReferenceProjection)' == '' and '$(AbiWinRTModernIdl)'!='false'">true</AbiWinRTEnableReferenceProjection>
+
+        <!-- Default ABI headers to always be prepended with the ABI namespace -->
+        <AbiWinRTAddAbiNamespacePrefix Condition="'$(AbiWinRTAddAbiNamespacePrefix)'==''">true</AbiWinRTAddAbiNamespacePrefix>
+        <AbiWinRTCommandAddAbiNamespacePrefix Condition="'$(AbiWinRTAddAbiNamespacePrefix)'=='true'">-ns-prefix always</AbiWinRTCommandAddAbiNamespacePrefix>
+
+        <!-- If not generating the platform headers, then default to ensure that there is compat -->
+        <AbiWinRTEnsureSDKHeaderCompat Condition="'$(AbiWinRTEnsureSDKHeaderCompat)'=='' and '$(AbiWinRTEnablePlatformProjection)'=='false'">true</AbiWinRTEnsureSDKHeaderCompat>
+        <AbiWinRTCommandEnsureSDKHeaderCompat Condition="'$(AbiWinRTEnsureSDKHeaderCompat)'=='true'">-lowercase-include-guard</AbiWinRTCommandEnsureSDKHeaderCompat>
+
+        <GeneratedFilesDir Condition="'$(GeneratedFilesDir)' == ''">$(IntDir)Generated Files\</GeneratedFilesDir>
+        <!--TEMP: Override NuGet SDK's erroneous setting in uap.props -->
```

Comment:
this may no longer be needed - test with nuget SDK distro (and/or ask @manodasanW )

Reply:
Doesn't look like it is